anvil/src/input_handler.rs

```diff
@@ -636,6 +646,8 @@ impl AnvilState<UdevData> {
                 },
             );
         }
+
+        // TODO send relative motion?
```

Comment:
No, we should not calculate relative-motion, either the device does give us relative events, or not.

Reply:
I was thinking it would need to do that to work on all backends, but I guess it can't really work if the backend doesn't provide relative pointer information.

Ideally it shouldn't advertise the protocol if it's not supported in the backend then, since the protocol won't actually work.

Reply:
> I was thinking it would need to do that to work on all backends, but I guess it can't really work if the backend doesn't provide relative pointer information.

Right. So once our wayland-backend is merged for example, we could extend it to support the relative-pointer protocol itself. But then that would also use the on_pointer_move function, I suppose? (Or the api would need some rework.)

> Ideally it shouldn't advertise the protocol if it's not supported in the backend then, since the protocol won't actually work.

Right, we could make it udev-backend specific in anvil for now. In the future that needs to be more dynamic though. E.g. support in the wayland-backend would depend on the host-compositor supporting the protocol as well.